### ISSUE TITLE
What is the current version of osmdroid?

### ISSUE THREAD

**Issue by mikebravoyes** [osmdroid/osmdroid#1946]
## Issue Type

[x] Question

## Description and/or steps/code to reproduce the problem

@spyhunter99 
What is the current version of osmdroid?

[https://github.com/osmdroid/osmdroid/releases](https://github.com/osmdroid/osmdroid/releases) says 6.1.17
That is also the version, which I use in build.gradle.
```
dependencies {
   ...
    implementation group: 'org.osmdroid', name: 'osmdroid-android', version: '6.1.17'
   ...
}
```

However, OsmdroidBuildInfo.java says 6.1.15:
```
package org.osmdroid;

//this file is automatically generated during the build
//when making changes, only edit ./src/main/filtered/...
//https://github.com/osmdroid/osmdroid/issues/1500
public class OsmdroidBuildInfo {

    private OsmdroidBuildInfo() {
    }

    public static final String VERSION = "6.1.15";
    public static final String BUILD_DATE = "Tue Mar 28 20:46:50 EDT 2023";
}

```
What is correct?

### Version of osmdroid the issue relates to:

6.1.15 ... 6.1.17 (?)

**Comment by spyhunter99** [comment]
that's unexpected. i'll look into it. latest version published to central
should be 6.1.17

**Comment by mikebravoyes** [comment]
@spyhunter99 
I just saw, that you have published the osmdroid version 6.1.18, good!
However, OsmdroidBuildInfo.java still says 6.1.15 (see above).

**Comment by spyhunter99** [comment]
sorry about that, forgot about this issue. it looks like it's a bug in gradle. It will only do the copy/filter if the destination does not exist, Which is contrary to the documentation and all the examples. It's entirely possible this started happening when we upgraded gradle after 6.1.15
looking at it now
